osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -29,6 +29,16 @@ public Aim(Mod[] mods)
         private double skillMultiplier => 23.25;
         private double strainDecayBase => 0.15;
 
+        private double calculateDefaultVelocity(double distance, double strainTime)
```

Comment:
Why are you buffing velocity below 90 bpm and nerfing everything above? If you think velocity isn't being calculated correctly it shouldn't be changed here, it should be changed in the difficulty object processing

Reply:
in OsuDifficultyHitObject, we should calculate it stupidly like 
`MovementDistance = calculateDefaultVelocity(MovementDistance, MovementTime) * MovementTime;`

in conversation, it is.
In fact, I think the yellow pattern is harder because we should locate the aim more longer.
imagine insane HDDT farming maps and 180bpm 6* None farming maps. (the two type of maps give same pp)